.github/workflows/minimal-tests-core.yml

```diff
@@ -69,6 +74,7 @@ jobs:
 
       # Setup Environments
       - name: Setup Environments
+        if: runner.os != 'Windows'
```

Comment:
We could possibly have an empty setup script for Windows, rather than skipping this step.